core/src/main/scala/com/pingcap/tispark/MetaManager.scala

```diff
@@ -23,7 +23,13 @@ import scala.collection.JavaConversions._
 
 // Likely this needs to be merge to client project
 // and serving inside metastore if any
-class MetaManager(catalog: Catalog) {
+class MetaManager(var catalog: Catalog) {
+
+  def reloadCatalog(newCatalog: Catalog): Unit = {
```

Comment:
Can you just reloadCache instead of reloadCatalog?

Reply:
I use a different catalog for stale read. so I need to update catalog here